desktop/fileManager.js

```diff
@@ -95,6 +95,39 @@ class FileManager {
     }
   };
 
+  static deleteFile = async ({ browserWindow, event, fileName, options, reloadOnDelete }) => {
+    try {
+      const filePath = path.isAbsolute(fileName) ? fileName : path.join(appFilesDirPath, fileName);
+      if (fs.existsSync(filePath)) {
```

Comment:
instead of late check for file existence make button disabled if no files are in store

Reply:
ok